packages/camels/camels.0.0.3/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+maintainer: ["maintenance@identity-function.com"]
+authors: ["yomimono"]
+homepage: "https://github.com/yomimono/camels"
+bug-reports: "https://github.com/yomimono/camels/issues"
+dev-repo: "git+https://github.com/yomimono/camels.git"
+license: "GPLv3"
+tags: ["game"]
+
+build: [
+  ["dune" "build"]
+]
+
+depends: [
+  "cmdliner"
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.8"}
```
Dune shouldn't be tagged as a build dependency (see #14266) and the `dune-project` file in the repository says this package requires at least dune 1.8